src/webgpu/api/validation/capability_checks/limits/maxBindGroupsPlusVertexBuffers.spec.ts

```diff
@@ -0,0 +1,274 @@
+/* eslint-disable prettier/prettier */
```

Comment:
Is there a particular thing in this file that needs this? Seems like pretty normal code.

Reply:
removed. I must have copy-pasta-ed something